akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -196,6 +196,12 @@ object Behavior {
     override def toString = "Unhandled"
   }
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi
+  private[akka] def failed[T](cause: Throwable): Behavior[T] = new FailedBehavior(cause).asInstanceOf[Behavior[T]]
```

Comment:
good, I don't think it should be public because that will only raise questions like: do I always have to catch exceptions and return failed? when to use failed, and when to throw?

Reply:
Okay let's leave this for itnernal use then